### PR TITLE
Fixes inhaler description

### PR DESCRIPTION
**Pull request by SonicGotNuked**
The descriptions all went off of the standard default item path. This fixes it.
[Screenshot from Gyazo](https://gyazo.com/4ad456acef313bff83e4529383328f08)
For example.

### CODE REVIEW

**Comment by SonicGotNuked** [comment]
!bugfix

**Comment by SonicGotNuked** [comment]
!review

**Comment by alsoandanswer** [comment]
you miss
you forgot these two.

File: code/modules/reagents/reagent_containers/inhaler_advanced.dm [hunk 1]
```diff
@@ -46,6 +46,7 @@
 
 /obj/item/weapon/reagent_containers/personal_inhaler_cartridge/large
 	name = "large inhaler cartridge"
+	desc = "A large inhaler cartridge. It contains enough areosal for 30 units of reagents. The container must be activated for aerosol to mix with reagents."
```
Comment (Arrow768) [inline code/modules/reagents/reagent_containers/inhaler_advanced.dm]:
```suggestion
	desc = "A large inhaler cartridge. It contains enough areosol for 30 units of reagents. The container must be activated for aerosol to mix with reagents."
```

File: code/modules/reagents/reagent_containers/inhaler_advanced.dm [hunk 2]
```diff
@@ -55,6 +56,7 @@
 
 /obj/item/weapon/reagent_containers/personal_inhaler_cartridge/bluespace
 	name = "bluespace inhaler cartridge"
+	desc = "An experimental bluespace inhaler cartridge. It has enough areosal for 60 units of reagents. The container must be activated to mix aerosol with reagents inside."
```
Comment (Arrow768) [inline code/modules/reagents/reagent_containers/inhaler_advanced.dm]:
```suggestion
	desc = "An experimental bluespace inhaler cartridge. It has enough areosol for 60 units of reagents. The container must be activated to mix aerosol with reagents inside."
```

**Pull request merged**